types/index.d.ts

```diff
@@ -71,6 +71,10 @@ declare namespace StyleDictionary {
     attributes: Attributes;
     path: string[];
     value: string;
+    /** A string representing the absolute path of the file that defines the token. */
```

Comment:
note: this should change based on https://github.com/amzn/style-dictionary/pull/499

Reply:
agreed- update as you see fit (you can look at what we did in #499 and update as you see fit)

Reply:
Updated thanks (and added descriptions for the additional keys that had docs added in #499).  Also fixed a typo in that file.